### DISCUSSION TITLE
付款后专业版仍然提示即将到期续费，请解决

### DISCUSSION THREAD

**Discussion by lb369lb** [getlantern/lantern-forum-cn#567] [General]
今天付款后，登录专业版，仍然提醒专业版到期需要续费。
当时我专业版的邮箱是：libo19840220@hotmail.com
付款凭证见下图，请帮忙解决一下，或者退款我重来一下。谢谢！
![7842805d2b48323a20ea817197225f3](https://github.com/getlantern/lantern-forum-cn/assets/51489267/a9f6a6d5-3858-4de0-8a53-222394c7fefe)
![146a1421ba4c4dec6ae0773ec6989e0](https://github.com/getlantern/lantern-forum-cn/assets/51489267/13493edf-dc76-4d78-b48e-6d20a19eb15a)

**Comment by MaxMKMax** [comment]
@lb369lb 您好. 请放心, 我帮您安排. 下周给您答复. 谢谢您的耐心等待.
